bin/jobsub_cmd

```diff
@@ -85,6 +87,33 @@ def main() -> None:
         if i.startswith("--"):
             i = i[1:]
 
+        if i in [
+            "-format",
+            "-xml",
+            "--json",
+            "-autoformat",
+            "-io",
+            "-batch",
+            "-nobatch",
+            "-dag",
+            "-long",
+        ]:
+            default_formatting = False
+
+        if (
+            i
+            in [
+                "-constraint",
+                "-factory",
+                "-autocluster",
+                "-constraint",
```

Comment:
I believe `-constraint` should not be in here twice.

Reply:
fixed.